kong-2.0.4-0.rockspec

```diff
@@ -20,7 +20,7 @@ dependencies = {
   "lua-ffi-zlib == 0.5",
   "multipart == 0.5.6",
   "version == 1.0.1",
-  "kong-lapis == 1.8.1.1",
+  "kong-lapis == 1.8.1.2",
```

Comment:
Was this bump part of a different PR?

Reply:
yup @bungle handled this elsewhere. It'll go away when I rebase